jaxlib/lapack.pyx

```diff
@@ -388,6 +388,14 @@ def jax_potrf(c, a, lower=False):
 
 # ?gesdd: Singular value decomposition
 
+cdef int gesdd_iwork_size(int m, int n) nogil:
+  return 8 * min(m, n)
+
+cdef int cgesdd_rwork_size(int m, int n) nogil:
+  cdef int mx = max(m, n)
+  cdef int mn = min(m, n)
+  return max(5 * mn * mn + 5 * mn, 2 * mx * mn + 2 * mn * mn + mn)
```

Comment:
You might consider adding a `7 * mn` term here because of this comment in the Lapack docs:
`LRWORK >= 5*mn (LAPACK <= 3.6 needs 7*mn)` for the `jobz = 'N'` case

Reply:
Isn't this expression an upper bound that would cover the 7 * mn case?

Reply:
Actually I came to the conclusion it is; 5*mn*mn + 5*mn is greater than 7*mn for all mn >= 0.

So fine either way I guess...